source/renderer/app/components/widgets/forms/InlineEditingInput.js

```diff
@@ -1,14 +1,23 @@
 // @flow
+/* eslint-disable react/no-did-update-set-state */
 import React, { Component } from 'react';
 import { observer } from 'mobx-react';
 import { defineMessages, intlShape } from 'react-intl';
+import { get } from 'lodash';
+import { Button } from 'react-polymorph/lib/components/Button';
+import { ButtonSkin } from 'react-polymorph/lib/skins/simple/ButtonSkin';
```

Comment:
@daniloprates You don't need to import and apply ButtonSkin.
It is used by default, so please remove this.